pkg/operator/ceph/object/objectstore.go

```diff
@@ -856,6 +892,8 @@ func enableRGWDashboard(context *Context) error {
 		return errors.Wrapf(err, "failed to create user %q", DashboardUser)
 	}
 
+	// TODO we should probably ensure that the access and secret keys for the dashboard-admin user
+	// are the same in every object store
```

Comment:
We would need to make sure this doesn't apply to external stores, however.

Reply:
So the current dashboard credentials should only be retrieved when the cluster is not external?

Reply:
I'm not sure if Rook can enable the dashboard for external clusters. @leseb do you know?

Reply:
at least from the code, there is no specific external cluster check for the dashboard service. IMO the current code does not consider the external cluster scenarios, high chance it won't work properly